fix: Update information for accessing supervisor (#1666)
appsmithorg/appsmith-docs: website/docs/getting-started/setup/instance-management/supervisor.md

APPSMITH_SUPERVISOR_PASSWORD=<SUPERVISOR-PASSWORD>
```

> 3. Use the interface to effectively manage the Appsmith server, Nginx, MongoDB, and other components. Access to the Supervisor web interface is available using the same domain on which Appsmith is operational. For example, if Appsmith is accessible via `http://localhost`, you can access the Supervisor web interface at `http://localhost/supervisor`.

Reviewer: ```suggestion
3. Access the web interface of Supervisord to manage the Appsmith server, Nginx, MongoDB, and other components at http://[my_domain]/supervisor. For example, if Appsmith is accessible via `http://localhost`, you can access the Supervisor web interface at `http://localhost/supervisor`.
```
